### ISSUE TITLE
Cocoapods: support for newest versions

### ISSUE THREAD

**Issue by netbe** [typelift/Swiftz#327]
Description
============

It seems Swiftz specs are not available in main repo of Cocoapods

```
pod search Swiftz
-> Swiftz (0.5.0)
   Swiftz is a Swift library for functional programming.
   pod 'Swiftz', '~> 0.5.0'
   - Homepage: https://github.com/typelift/Swiftz
   - Source:   https://github.com/typelift/Swiftz.git
   - Versions: 0.5.0, 0.4.0, 0.3.1, 0.3.0, 0.2.4, 0.2.2 [master repo]
```

I see that Cocoapods is not mentionned in Readme as setup, so I am just wondering if it is supported or not

Expected Result
================

Upload missing versions to [Cocoapods trunk](https://guides.cocoapods.org/making/getting-setup-with-trunk.html#deploying-a-library)

**Comment by wpoosanguansit** [comment]
Hi,

I am using pod to install the lib but it does not look like it is usable yet.  From what I can see, not all the code is being brought in.  I could not get the code in the intro to compile at all.  Thanks.


![image](https://user-images.githubusercontent.com/90733/51804784-e00d3700-2219-11e9-9cd9-78a37a2e6085.png)

**Comment by CodaFi** [comment]
Yeah, this is a problem on CocoaPods' end related to code signing - I would have loved to have pushed to their trunk but I cannot get the pod to validate.  The 1.6 beta does work under some circumstances so I'll give that a try this evening.
